salt/utils/smb.py

```diff
@@ -426,7 +426,7 @@ def _delete_directory_smbprotocol(path, share='C$', conn=None, host=None, userna
         conn = get_conn(host, username, password)
     if conn is False:
         return False
-    log.warn("PATH: %s %s", share, path)
+    log.warning("PATH: %s %s", share, path)
```

Comment:
Path for what? Either this needs to be a debug (and then we know its context) or rephrased with more expanded explanation and a context.

Reply:
I touched these because it was polluting my output (seeing warnings about warnings :) ).

I can try to rephrase some of them but that should be reviewed, since I'm really out of context for all of them.

Reply:
@rares-pop we can also do the other PR though. But it wouldn't hurt if you would offer less work for others. :wink: 

I leave it open to you.

Reply:
I meant, I will change the warning messages in this PR, but you or someone who has more/better context should double-check that they make sense after rephrasing, since I'm not really familiar with the code-base.

Reply:
You bet! OK, you've getting my :x: now again. :laughing: